presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestEvaluateZeroLimit.java

```diff
@@ -33,8 +33,7 @@ public void testDoesNotFire()
                 .on(p ->
                         p.limit(
                                 1,
-                                p.values(p.variable("a"))))
-                .doesNotFire();
+                                p.values(p.variable("a"))));
```

Comment:
can you share more about this change? Also, it should be in the previous commit.

Reply:
this was left out unintensionally. added back.